src/test/appForm.test.js

```diff
@@ -374,4 +385,126 @@ describe("App Form", function () {
 
   });
 
+  describe("on server response errors", function () {
+
+    before(function (done) {
+      this.server = server
+      .setup({}, 200)
+      .start(done);
+    });
+
+    after(function (done) {
+      this.server.stop(done);
+    });
+
+    describe("App form store", function () {
+
+      it("processes array of details on code 400", function (done) {
+        this.server.setup({
+          details: [{
+            "path": "/instances",
+            "errors": [
+              "error.expected.jsnumber"
+            ]
+          }]
+        }, 400);
+
+        AppsStore.once(AppsEvents.CREATE_APP_ERROR, function () {
+          expectAsync(function () {
+            expect(AppFormStore.responseErrors.instances)
+              .to.equal("error.expected.jsnumber");
+          }, done);
+        });
+
+        AppsActions.createApp({
+          instances: "many"
+        });
+      });
+
+      it("processes array of errors on code 422", function (done) {
```

Comment:
let's pick better wording if possible, see my suggestion above (same applies for following describes)